docs/build-instructions/windows.md

```diff
@@ -48,6 +48,17 @@ This will create the Atom application in the `out\Atom` folder as well as copy i
   * `--no-install` - Skips the installation task after building.
   * `--verbose` - Verbose mode. A lot more information output.
 
+### Running tests
+
+In order to run tests from command line you need `apm`, available after you install Atom or after you build that from scratch. If you installed it, run the following commands (assuming `C:\atom` is the root path of your Atom sources):
+
+```bash
+cd C:\atom
+apm test
+```
+
+When building Atom from scratch instead, the `apm` command is not added to the system path by default. In this case, you can either add it yourselves or you can explicitly list the complete path in previous commands. Anyway, its default location on a Win 64-bit installation is `"C:\Program Files (x86)\Atom\resources\cli\"`.
```

Comment:
* building Atom from scratch instead &rarr; building Atom from source
* add it yourselves &rarr; add it yourself
* or you can explicitly &rarr; or explicitly